index.js

```diff
@@ -439,6 +447,12 @@ Converter.prototype.buildParameter = function(oldParameter) {
       'Complex type is used inside non-body argument.');
   }
 
+  //According to Swagger 2.0 spec: If the parameter is in "path",
+  //this property is required and its value MUST be true.
+  if (parameter.in === 'path') {
```

Comment:
See, that what happens when I don't spend enough time on things :) Anyways, right you are, I'll open a ticket on swagger-spec to fix it in the schema. It *is* required though, just a bug in the schema.